shpc/main/client.py

```diff
@@ -61,6 +61,15 @@ def install(self, name, tag=None, **kwargs):
         Install must be implemented by the subclass (e.g., lmod)
         """
         raise NotImplementedError
+    
+    def reinstall(self, name, update_containers=False):
```

Comment:
Here it should raise NotImplementedError, and the actual implementation of the function moved to ModuleBase

Reply:
Ignore that, let Vanessa give her view